### PR TITLE
    update deploy logic and make  arc and setup-env docs migration

### PR DESCRIPTION
**Pull request by imlidian**

### CODE REVIEW

File: _users/cn/servicecomb-introduction.md [hunk 1]
```diff
@@ -1,38 +1,21 @@
...
 
 {% include toc %}
-## 框架概述
-![ServiceComb Model](/assets/images/servicecomb-models.png){: .align-center}
 
-## 主要设计意图
+## [Java Chassis](/users/servicecomb-java-chassis-doc/zh_CN/){:target="_blank"}
+• ⾼性能的微服务框架，提供服务注册发现以及动态配置管理，以及限流降级，容错熔 断等服务治理功能  
+## [Service Center](https://github.com/apache/incubator-servicecomb-saga){:target="_blank"}  
+• 基于Etcd⾼性能，⾼可⽤服务注册中⼼
```
Comment (zhengyangyong) [inline _users/cn/servicecomb-introduction.md]: 乱码？

File: _users/cn/servicecomb-introduction.md [hunk 2]
```diff
@@ -1,38 +1,21 @@
...
 ---
 
 {% include toc %}
-## 框架概述
-![ServiceComb Model](/assets/images/servicecomb-models.png){: .align-center}
 
-## 主要设计意图
+## [Java Chassis](/users/servicecomb-java-chassis-doc/zh_CN/){:target="_blank"}
+• ⾼性能的微服务框架，提供服务注册发现以及动态配置管理，以及限流降级，容错熔 断等服务治理功能  
+## [Service Center](https://github.com/apache/incubator-servicecomb-saga){:target="_blank"}
```
Comment (WillemJiang) [inline _users/cn/servicecomb-introduction.md]: service-center对上saga。

File: _users/cn/servicecomb-introduction.md [hunk 3]
```diff
@@ -1,38 +1,21 @@
...
-| 编程模型 | provider-pojo | 是 | 提供RPC开发模式 |
-| 编程模型 | provider-jaxrs | 是 | 提供JAX RS开发模式 |
-| 编程模型 | provider-springmvc | 是 | 提供Spring MVC开发模式 |
-| 通信模型 | transport-rest-vertx | 是 | 运行于HTTP之上的开发框架，不依赖WEB容器，应用打包为可执行jar |
-| 通信模型 | transport-rest-servlet | 是 | 运行于WEB容器的开发框架，应用打包为war包 |
-| 通信模型 | transport-highway | 是 | 提供高性能的私有通信协议，仅用于java之间互通。 |
-| 运行模型 | handler-loadbalance | 是 | 负载均衡模块。提供各种路由策略和配置方法。一般客户端使用。 |
-| 运行模型 | handler-bizkeeper | 是 | 和服务治理相关的功能，比如隔离、熔断、容错。 |
-| 运行模型 | handler-tracing | 是 | 调用链跟踪模块，对接监控系统，吐出打点数据。 |
+## [Saga](https://github.com/apache/incubator-servicecomb-service-center){:target="_blank"}
```
Comment (WillemJiang) [inline _users/cn/servicecomb-introduction.md]: 这块链接也不对，saga对上service center了。

File: script/deploy [hunk 4]
```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
-
+script_dir=$(cd `dirname $0`; pwd)
+echo cru$script_dir
+base_dir=`dirname ${script_dir}`
+echo build $base_dir
+echo $git_book_dir
 git clone https://github.com/apache/incubator-servicecomb-docs.git
```
Comment (WillemJiang) [inline script/deploy]: It looks the script won't check the change the servicecomb doc, and just rebuild the whole document once the site doc is changed.

Comment (imlidian) [reply script/deploy]: cancel the gitbook building logic

File: script/deploy [hunk 5]
```diff
@@ -1,10 +1,9 @@
...
-bundle exec jekyll build --config _config.yml
-
-bundle exec jekyll server --port 4000 --host 
+script_dir=$(cd `dirname $0`; pwd)
+echo cru$script_dir
+base_dir=`dirname ${script_dir}`
+echo build $base_dir
+cd ${base_dir}
+sudo bundle exec jekyll build --config _config.yml
+sudo bundle exec jekyll server --port 4000 --host
```
Comment (WillemJiang) [inline script/deploy]: when running the server the host url is changed,  it is not for the asf-site to use.

File: _users/cn/servicecomb-introduction.md [hunk 6]
```diff
@@ -1,38 +1,21 @@
...
 redirect_from:
   - /theme-setup/
 ---
 
 {% include toc %}
-## 框架概述
-![ServiceComb Model](/assets/images/servicecomb-models.png){: .align-center}
 
-## 主要设计意图
+## [Java Chassis](/users/servicecomb-java-chassis-doc/zh_CN/){:target="_blank"}
```
Comment (WillemJiang) [inline _users/cn/servicecomb-introduction.md]: Please update the document url to the latest one on the navigate site.

**Pull request closed without merging**